README.md

```diff
@@ -13,6 +13,17 @@ If you want to contribute projects or tools, please first read our [Contributing
 
 Submit a pull request to add a link to your repository here.
 
+### Tungsten demo day
```

Comment:
```suggestion
### Demo day projects
```
I really recommend de-emphasizing the use of internal milestone names--they suggest multiple parallel networks and really only create confusion (for example, "tungsten" isn't different from"sodium" or "mercury").